rst2pdf/extensions/fancytitles.py

```diff
@@ -1,14 +1,14 @@
 # -*- coding: utf-8 -*-
+import codecs
+import tempfile
+
```

Comment:
General question: What's the style rule that determines why this blank line in the imports list is here?

Reply:
It's import ordering. From PEP8:

> Imports should be grouped in the following order:
>
> Standard library imports.
> Related third party imports.
> Local application/library specific imports.
>
> You should put a blank line between each group of imports.

flake8 won't check these for us since it's out of scope, but other tools like isort can

Reply:
Thanks. Ironically, I came across this just now when looking up up relative imports.